src/foam/java/refinements.js

```diff
@@ -34,6 +34,11 @@ foam.CLASS({
       name: 'javaJSONParser',
       value: 'foam.lib.json.AnyParser.instance()'
     },
+    {
+      class: 'String',
+      name: 'javaQueryParser',
+      value: 'foam.parse.AnyParser.instance()'
```

Comment:
Instead of value: 'foam.parse.AnyParser.instance()' let's use expression: function(javaJSONParser) { return javaJSONParser; }

This way the default javaQueryParser is the same as the JSON parser, so we don't have to specify it if it's the same.